### PR TITLE
Create a stub of the kevinmalhotra kincardine file

### PR DESCRIPTION
**Pull request by kevinmalhotra**
Adding the hometown details

### CODE REVIEW

**Comment by francisco-cho** [comment]
I can't believe you didn't tag me :(

File: kevinmalhotra-kincardine.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Kincardine, ON
```
Comment (francisco-cho) [inline kevinmalhotra-kincardine.md]: sounds boring

File: kevinmalhotra-kincardine.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+### Places to Visit
```
Comment (nathyga) [inline kevinmalhotra-kincardine.md]: Where can I go?

File: kevinmalhotra-kincardine.md (hunk 1 above)
Comment (kevinmalhotra) [inline kevinmalhotra-kincardine.md]: It's only boring if you hate beaches, pipe bands, and tranquility.

File: kevinmalhotra-kincardine.md (hunk 2 above)
Comment (kevinmalhotra) [inline kevinmalhotra-kincardine.md]: You can take a beach walk for an hour!

File: kevinmalhotra-kincardine.md (hunk 1 above)
Comment (francisvicta) [inline kevinmalhotra-kincardine.md]: I like Wasaga.

**Comment by kevinmalhotra** [comment]
@nathyga

**Comment by kevinmalhotra** [comment]
@francisco-cho

**Comment by kevinmalhotra** [comment]
@francisvicta

File: kevinmalhotra-kincardine.md (hunk 1 above)
Comment (kevinmalhotra) [inline kevinmalhotra-kincardine.md]: Wasaga Beach is for car-campers who don't like to shower.

**Pull request merged**